xbdd-web/src/components/BuildHistoryView.jsx

```diff
@@ -0,0 +1,69 @@
+
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import Tooltip from '@material-ui/core/Tooltip';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
+import BlockIcon from '@material-ui/icons/Block';
+import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import GridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import { Card } from '@material-ui/core';
+import FeatureHistory from '../models/FeatureHistory';
+
+const styles = theme => ({
+    buildHistoryViewContainer: {
+        width: '100%',
+        maxWidth: 450,
+        backgroundColor: theme.palette.background.paper,
+    },
+});
+
+const getIcon = (status, lastEditedBy) => {
```

Comment:
This could potentially be moved into its own component? I think it has use cases beyond this component, like ScenarioListView to display the icon